index.html

```diff
@@ -1808,78 +1877,87 @@ <h2>Authentication</h2>
       </p>
 
-      <p class="note" title="Authorization vs authentication">
-Note that Authorization is separate from <a href="#authentication"></a>.
-This is particularly important for key recovery in the case of key loss,
-when the subject no longer has access to their keys, or key compromise,
-where the <a>DID controller</a>'s trusted third parties need to override
-malicious activity by an attacker. See Section
-<a href="#security-considerations"></a>.
-      </p>
+      <p class="note" title="Uses of assertionMethod">
+If assertionMethod is established, it is up to the verifying party to validate that the <a>verification method</a> used for providing proof of an assertion is valid and is contained in the assertionMethod section. An example of how this might be used, is when a verifier validates a verifiable credential has been signed on behalf of the <a>DID Subject</a> to issue a verifiable credential. In this case, the verifier would want to check the <a>verification method</a> used to assert proof on behalf of the <a>Did Subject</a> is contained within the assertionMethod section of the <a>DID Document</a>.
 
       <p>
-A DID document MAY include a <code>controller</code> property to indicate
-that there are <a>DID controller(s)</a> other than the <a>DID subject</a>.
-If so:
-      </p>
-
-      <dl>
-          <dt><dfn>controller</dfn></dt>
-          <dd>
-The value of the <code>controller</code> property MUST be a valid <a>DID</a>
-or an array of valid <a>DIDs</a>. The corresponding <a>DID document</a>(s)
-SHOULD contain authorization relations that explicitly permit the use of
-certain verification methods for specific purposes.
-          </dd>
-      </dl>
+A <a>DID document</a> MAY include an <code>assertionMethod</code> property.
+The <code>assertionMethod</code> property is an example of a <a>verification
+relationship</a>.
 
       <p>
-Each <a>DID method</a> MUST define how authorization and delegation are
-implemented, including any necessary cryptographic operations.
+Example:
       </p>
 
+      <pre class="example nohighlight" title="Authentication field containing three verification methods">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "assertionMethod": [
+    <span class="comment">// this method can be used to assert statements as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method can be used to assert statements as did:...fghi</span>
+    "did:example:123456789abcdefghi#biometric-1",
```

Comment:
biometric is a strange example of an assertion method, we might want to remove it?

Reply:
Was just copied from the authentication section. Will remove it as it's not necessary to highlight the purpose of this section of the document.